aspnetcore/data/ef-mvc/read-related-data.md

```diff
@@ -209,7 +209,7 @@ In the *Views/Instructors/Index.cshtml* file, after the closing table element (a
 
 This code reads the `Courses` property of the view model to display a list of courses. It also provides a **Select** hyperlink that sends the ID of the selected course to the `Index` action method.
 
-Run the page and select an instructor. Now you see a grid that displays courses assigned to the selected instructor, and for each course you see the name of the assigned department.
+Refresh the page and select an instructor. Now you see a grid that displays courses assigned to the selected instructor, and for each course you see the name of the assigned department.
```

Comment:
Used refresh here because the app is already running and we've just changed the .cshtml files which don't require re-running the app to take effect AFAIK...